plugins/quay-common/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "@janus-idp/backstage-plugin-quay-common",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
```

```suggestion
  "license": "Apache-2.0",
  "private": true,
```

At the moment, we are unable to release new plugins through GitHub. So what we have to do is set this to private so that it is skipped in the release process and then I can go in and manually release this plugin and update the dependencies.